### ISSUE TITLE
💻 Replit compatibility

### ISSUE THREAD

**Issue by MumblesUsed2MakeBeats** [reworkd/AgentGPT#54]
Would be great to run this on Replit mobile app

**Comment by asim-shrestha** [comment]
Yeah that was something we were considering, I don't think they have an API we can interact with however. If anyone knows how this might get accomplished, that would be awesome

**Comment by kenorb** [comment]
Somebody did demo: https://twitter.com/chillzaza_/status/1645923169654505473
